setup/06-regional-by-row/movr/users/data/users_database.sql

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE movr_users;
+
+CREATE TABLE movr_users.users (
+	email STRING PRIMARY KEY,
+	city STRING NULL,
+	last_name STRING NOT NULL,
+	first_name STRING NOT NULL,
+	phone_numbers STRING[] NOT NULL
+);
```

Comment:
```suggestion
);

ALTER DATABASE movr_users SET PRIMARY REGION "us-east";
ALTER DATABASE movr_users ADD REGION "us-central";
ALTER DATABASE movr_users ADD REGION "us-west";
```

The question in the instructions assumes that the regions have been added